### PR TITLE
[chore](pipeline) optimize profile information

### PR DESCRIPTION
**Pull request by mrhhsg**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 37.06 seconds
 load time: 661 seconds
 storage size: 17123782694 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221228092642_clickbench_pr_70178.html

File: be/src/pipeline/exec/assert_num_rows_operator.h [hunk 1]
```diff
@@ -36,6 +36,7 @@ class AssertNumRowsOperator final : public StreamingOperator<AssertNumRowsOperat
 public:
     AssertNumRowsOperator(OperatorBuilderBase* operator_builder, ExecNode* node)
             : StreamingOperator(operator_builder, node) {};
+    std::string get_name() const override { return "AssertNumRowsOperator"; }
```
Comment (HappenLee) [inline be/src/pipeline/exec/assert_num_rows_operator.h:39]: do not need override each opeator. just use opeartor build `get_name` fucntion.

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**